.github/workflows/check-pr-title.yaml

```diff
@@ -8,8 +8,7 @@ on:
       - synchronize
 
 jobs:
-  check-pr-title:
-    name: Check PR Title
+  semantic-commit:
```

Comment:
not sure what semantic-commit is

Reply:
this should be [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/), because that is what it is checking.